app/Dockerfile

```diff
@@ -0,0 +1,79 @@
+FROM ubuntu:20.04 AS base
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+
+RUN apt-get update -y && \
```

Comment:
minor: one could consider pinning versions in the future, although I myself often dont do this either /o\